frontend/styles/Login.module.css

```
@@ -0,0 +1,75 @@
+.background {
+  padding: 0;
+  margin: 0;
+  height: 100vh;
+  width: auto;
+  background-color: #f2f2f2;
+}
+
+.login-form {
+  width: 500px;
+  height: auto;
+  margin: 5% auto;
+  display: flex;
+  flex-direction: column;
+  background-color: white;
+  border-radius: 1rem;
+  padding: 20px 20px 20px 20px;
+}
+
+.title {
+  font-size: 1.3rem;
+  padding: 0;
+  margin: 0;
+}
+
+.login {
+  -webkit-appearance: none;
+  width: 30%;
+  min-width: 100px;
+  border-radius: 24px;
+  text-align: center;
+  padding: 15px 40px;
+  margin-top: 5px;
```

It would be nice to center this button along with the headers:
```suggestion
  margin: 5px auto 0;
```